daisy_integration_tests/v2v_adapt_windows.wf.json

```diff
@@ -0,0 +1,93 @@
+{
+  "Name": "v2v-adapt",
+  "ForceCleanupOnError": true,
+  "Vars": {
+    "image_name": {
+      "Required": true,
+      "Description": "The name of the imported image."
+    },
+    "source_disk": {
+      "Required": true
+    },
+    "worker_disk": {
+      "Value": "worker-disk"
+    }
+  },
+  "Steps": {
+    "create-disk": {
+      "CreateDisks": [
+        {
+          "Name": "${worker_disk}",
+          "SourceImage": "projects/v2v-dev/global/images/family/v2v-candidate",
+	        "SizeGb": "200",
+          "Type": "pd-standard",
+          "Licenses": [
+            "https://compute.googleapis.com/compute/v1/projects/vm-options/global/licenses/enable-vmx"
+          ]
+        }
+      ]
+    },
+    "create-instance": {
+      "CreateInstances": [
+        {
+          "Name": "worker",
+          "Disks": [
+            {
+              "Source": "${worker_disk}",
+              "DeviceName": "disk"
+            },
+            {
+              "Source": "${source_disk}",
+              "DeviceName": "workload-disk1"
+            }
+          ],
+          "MachineType": "n1-standard-2",
+          "Metadata": {
+            "block-project-ssh-keys": "true",
+            "enable-guest-attributes": "true",
+            "startup-script": "docker run --privileged -v /dev/disk/by-id:/dev/disk/by-id:ro v2v -vv --log file=/dev/ttyS1,level=debug --log file=/dev/ttyS2,level=trace --report file=/dev/ttyS3 --report file=http://metadata.google.internal/computeMetadata/v1/instance/guest-attributes/v2v/report,headers=Metadata-Flavor=Google --drive file=/dev/disk/by-id/google-workload-disk1 --in-place"
```

Comment:
To make this more readable, while retaining the inline format, you can put it in its own file, and use "source vars" as described here:

https://github.com/GoogleCloudPlatform/compute-image-tools/blob/master/docs/daisy-workflow-config-spec.md

Reply:
Extracted to a file